v2.1/configure-replication-zones.md

```diff
@@ -826,70 +497,88 @@ There's no need to make zone configuration changes; by default, the cluster is c
+    +-----------+------------------------------------------+
+      .default  | ALTER RANGE default CONFIGURE ZONE USING
+                |     range_min_bytes = 1048576,
+                |     range_max_bytes = 67108864,
+                |     gc.ttlseconds = 90000,
+                |     num_replicas = 5,
+                |     constraints = '[]',
+                |     lease_preferences = '[]'
+    (1 row)
     ~~~
 
     All data in the cluster will be replicated 5 times, including both SQL data and the internal system data.
 
-3. On any node, configure the `.meta` replication zone:
+5. Configure the `.meta` replication zone:
 
     {% include copy-clipboard.html %}
-   ~~~ shell
-   echo 'num_replicas: 7' | cockroach zone set .meta \
-   --insecure \
-   --host=<any node hostname> \
-   -f -
-   ~~~
-
-   ~~~
-   range_min_bytes: 1048576
-   range_max_bytes: 67108864
-   gc:
-     ttlseconds: 86400
-   num_replicas: 7
-   constraints: []
-   ~~~
+    ~~~ sql
+    > ALTER RANGE meta CONFIGURE ZONE USING num_replicas = 7;
+    ~~~
+
+    ~~~
+      zone_name |              config_sql
+    +-----------+---------------------------------------+
+      .meta     | ALTER RANGE meta CONFIGURE ZONE USING
+                |     range_min_bytes = 1048576,
+                |     range_max_bytes = 67108864,
+                |     gc.ttlseconds = 3600,
+                |     num_replicas = 7,
+                |     constraints = '[]',
+                |     lease_preferences = '[]'
+    (1 row)
+    ~~~
 
    The `.meta` addressing ranges will be replicated such that one copy is in all 7 datacenters, while all other data will be replicated 5 times.
 
-4. On any node, configure the `.timeseries` replication zone:
+6. Configure the `.timeseries` replication zone:
 
     {% include copy-clipboard.html %}
-   ~~~ shell
-   echo 'num_replicas: 3' | cockroach zone set .timeseries \
-   --insecure \
-   --host=<any node hostname> \
-   -f -
-   ~~~
-
-   ~~~
-   range_min_bytes: 1048576
-   range_max_bytes: 67108864
-   gc:
-     ttlseconds: 86400
-   num_replicas: 3
-   constraints: []
-   ~~~
+   ~~~ sql
+   > ALTER RANGE timeseries CONFIGURE ZONE USING num_replicas = 3;
+    ~~~
+
+    ~~~
+       zone_name  |                 config_sql
+    +-------------+---------------------------------------------+
+      .timeseries | ALTER RANGE timeseries CONFIGURE ZONE USING
+                  |     range_min_bytes = 1048576,
+                  |     range_max_bytes = 67108864,
+                  |     gc.ttlseconds = 90000,
+                  |     num_replicas = 3,
+                  |     constraints = '[]',
+                  |     lease_preferences = '[]'
+    (1 row)
+    ~~~
 
    The timeseries data will only be replicated 3 times without affecting the configuration of all other data.
 
 ## See also
 
-- [Other Cockroach Commands](cockroach-commands.html)
+- [`SHOW ZONE CONFIGURATIONS`](show-zone-configurations.html)
```

Comment:
Not sure why (possibly related to the rendering issue above), but the *See Also* section is not rendering at all for me (see the bottom of http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/3059e70601731e0ec788e40c69e7a9ab6e06dc4b/dev/configure-replication-zones.html).